kotlin/src/main/kotlin/org/axonframework/extensions/kotlin/QueryGatewayExtensions.kt

```diff
@@ -25,29 +25,22 @@ import java.util.*
 import java.util.concurrent.CompletableFuture
 
 /**
- * Extensions to access query gateway.
+ * Reified version of [QueryGateway.queryForMultiple].
```

Comment:
I think it's an extension of the `QueryGateway.query`, not `QueryGateway.queryForMultiple`. It defaults to a multiple instances of, that's true, but that does not change the origination point I think.